math/fricas/Portfile

```diff
@@ -28,10 +28,10 @@ distname            ${name}-${version}-full
 use_bzip2           yes
 worksrcdir          ${name}-${version}
                 
-checksums           rmd160  1ab5c148f0684c946c3b0831a5fe01fc21bbe83a \
-                    sha256  798b123e61f3566e6b1b367c235f5b4599a9c737b938aa71f32d0aabb7db439d \
-                    md5     0d3af65758ce93b1cc52c2511e73e674 \
-                    sha1    8fc3e850a9890eac21cd9f391dd58000b3537067
+checksums           rmd160  10aac421e8100668b2e6a9a598da353290ca354e \
+                    sha256  fd94bb0ff5f4de1dc10eef37446a6e7b5bf14d61852089002a7e0813be4ad394 \
+                    md5     cef44e511d124b1f6e1f6b5ac53fd496 \
+                    sha1    3e84e978fd5ba0b269890b7acfdad9ba3d380c31
```

Comment:
We no longer include md5 or sha1 hashes in our Portfiles. However, we now _do_ include a `size` field. If you can add that, and remove the obsolete hashes, I'll commit this very quickly.